doc/dev/TESTING.md

```diff
@@ -1,77 +1,117 @@
 # Testing
 
-F3D has an expansive suite of tests, you may want to run them locally,
-either to validate your build or because you are contributing to F3D and want to add/modify a test.
+F3D has an expansive suite of tests that can be run locally, either to validate your build or because you are contributing to F3D and want to add/modify a test.
+To enable the tests, run `cmake -DBUILD_TESTING=ON ../` from the `build`  directory.
 
 ## CMake Options
 
-There is a few CMake options to F3D testing configuration:
-* `BUILD_TESTING`: Enable the tests, off by default.
+There are a few CMake options to configure the F3D testing framework:
 * `F3D_TESTING_ENABLE_RENDERING_TESTS`: An option to enable/disable test that require rendering capabilities, on by default.
 * `F3D_TESTING_ENABLE_LONG_TIMEOUT_TESTS`: Certain tests can take some time to run, off by default, requires rendering tests.
 * `F3D_TESTING_DISABLE_DEFAULT_LIGHTS_TESTS_COMPARISON`: With VTK < 9.0.0, rendering can be very different, although not incorrect, so this option is provided, off by default.
 
 ## Running the tests
 
-To run the tests, just use ctest from the build directory:
+To run all tests, build the `f3d` binary with `make` and then run `ctest` from the build directory:
```

Comment:
"build and then run", no need to specify the f3d executable imo.

Reply:
Noted